websites/K/KaniNetwork/presence.ts

```diff
@@ -4,40 +4,42 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
-      largeImageKey: "logo"
+      largeImageKey: "logo",
+      smallImageKey: "logo",
+      smallImageText: "KaniShiel"
```

Comment:
and maybe delete it then if you don't have something different for smallImage :/